src/App.tsx

```diff
@@ -1,16 +1,23 @@
 import './App.scss';
+import { useState } from 'react';
+
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
 import moviesFromServer from './api/movies.json';
 
 export const App = () => {
+  const [movies, setMovies] = useState(moviesFromServer);
+
   return (
     <div className="page">
       <div className="page-content">
-        <MoviesList movies={moviesFromServer} />
+        <MoviesList movies={movies} />
       </div>
       <div className="sidebar">
-        <NewMovie /* onAdd={(movie) => {}} */ />
+        <NewMovie onAdd={(newMovie) => {
+          setMovies([...movies, newMovie]);
+        }}
+        />
```

Comment:
better to create separate function
```suggestion
        <NewMovie onAdd={onAdd} />
```